docs/custom-flows/sign-out.mdx

```diff
@@ -95,13 +105,17 @@ Please note that the sign-out flow only deactivates the current session. Other v
               type="text/javascript"
             ></script>
 
+            <!-- Your custom sign-out button -->
             <script>
               window.addEventListener('load', async function () {
                 await Clerk.load()
 
                 if (Clerk.user) {
-                  document.getElementById('sign-out').addEventListener('click', async () => {
+                  // Attach signOut function to the sign-out button
```

Comment:
Be wary of unnecessary comments - a developer should know what this line is doing

```suggestion
```